.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: goreleaser
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+jobs:
+  name: Release a new version
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13
+        id: go
+
+      - name: Install hub
+        uses: geertvdc/setup-hub@master
+
+      - name: Update AWS cli
+        uses: chrislennon/action-aws-cli@v1.1
+
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\//}
+
+      - name: Get the builtvar
+        id: get_built
+        run: echo ::set-output name=BUILT::$(date -u +%a_%d_%b_%H:%M:%S_%Y)
+
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN_EXTRA }}
+          VERSION: ${{ steps.get_version.outputs.VERSION }}
+          BUILT: ${{ steps.get_built.outputs.BUILT }}
+          OWNER: elastic
+          REPO: ecctl
+        with:
+          version: latest
+          args: release --rm-dist
+
+      - name: Run release post actions
+        run: ./scripts/goreleaser-post-actions.sh $VERSION
+        env: 
+          GITHUB_TOKEN: ${{ secrets.GH_TOKEN_EXTRA }}
+          GITHUB_USER: marclop
```

Comment:
Is it possible to use `elastic`, the tooling team or each of us use our own usernames?

Reply:
So, the trouble here is that the user name needs to match the username which created the `GH_TOKEN_EXTRA` which is used to fork and create the Pull Request too, that's why. 

In time I think a user with permissions to our repos which can be used for these sort of things is best.

This is a small step towards that direction

Reply:
I have the same issue in the soteria project where I'm using a personal token with extra permissions